data/actions.js

```diff
@@ -8,7 +8,7 @@ function hideKeyboard () {
 
 function validateEmail (email) {
 	// return true if invalid
-	return email !== '' && !email.match(/^[-a-z0-9~!$%^&*_=+}{\'?]+(\.[-a-z0-9~!$%^&*_=+}{\'?]+)*@([a-z0-9_][-a-z0-9_]*(\.[-a-z0-9_]+)*\.(aero|arpa|biz|com|coop|edu|gov|info|int|mil|museum|name|net|org|pro|travel|mobi|[a-z][a-z])|([0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}))(:[0-9]{1,5})?$/i)
+	return email !== '' && !email.match(/^[-a-z0-9~!$%^&*_=+}{\'?]+(\.[-a-z0-9~!$%^&*_=+}{\'?]+)*@([a-z0-9_][-a-z0-9_]*(\.[-a-z0-9_]+)*\.(aero|arpa|biz|com|coop|edu|email|gov|info|int|mil|museum|name|net|org|pro|travel|mobi|[a-z][a-z])|([0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}))(:[0-9]{1,5})?$/i)
```

Comment:
i think we should probably just make this much more relaxed………

there are so many valid domains nowadays:-
https://iwantmyname.com/?domain=benfletcher

<img width="209" alt="screen shot 2016-06-14 at 11 46 09" src="https://cloud.githubusercontent.com/assets/825088/16040105/9fb23d9a-3225-11e6-810c-2ef0292488ef.png">